### ISSUE TITLE
Terminating OpenVPN process makes gotop exit

### ISSUE THREAD

**Issue by egeesin** [cjbassi/gotop#38]
* gotop -v: 1.4.0
* uname -a: Linux ege-arch 4.16.10-1-macbook #1 SMP PREEMPT Mon May 21 20:51:39 EDT 2018 x86_64 GNU/Linux
* Shell: zsh
* Emulator: Alacritty
* No tmux

When I terminate openvpn, gotop gave this output:
```
Error caught. Exiting program.
Issue with net data.
Diagnostics:
curRecvTotal: 698817528
curSentTotal: 86045375
self.prevRecvTotal: 706101718
self.prevSentTotal: 86246367
recvRecent: 18446744073702267426
sentRecent: 18446744073709350624
int(recvRecent): -7284190
int(sentRecent): -200992
panic: 1
goroutine 54 [running]:
github.com/cjbassi/gotop/src/utils.Error(0x58301f, 0x8, 0xc4200220e0, 0xde)
    /home/cjbassi/go/src/github.com/cjbassi/gotop/src/utils/utils.go:49 +0x23f
github.com/cjbassi/gotop/src/widgets.(*Net).update(0xc420264fa0)
    /home/cjbassi/go/src/github.com/cjbassi/gotop/src/widgets/net.go:60 +0x519
github.com/cjbassi/gotop/src/widgets.NewNet.func1(0xc4201d22d0, 0xc420264fa0)
    /home/cjbassi/go/src/github.com/cjbassi/gotop/src/widgets/net.go:39 +0x5c
created by github.com/cjbassi/gotop/src/widgets.NewNet
    /home/cjbassi/go/src/github.com/cjbassi/gotop/src/widgets/net.go:37 +0x20e
```

**Comment by cjbassi** [comment]
This is a similar issue to #12, and is caused by an issue with gopsutil. I've already created an issue at the gopsutil repo for #12 so I'll just link them to this one too from the same issue.

**Comment by cjbassi** [comment]
Should be fixed with c7171781c13f2bdd2f3cffd149800426a97dd4c2. Let me know if you still have issues.
